i18n/miraheze/en.json

```diff
@@ -95,30 +95,28 @@
 	"grouppage-steward": "m:Stewards",
 	"group-sysadmin": "System administrators",
 	"group-sysadmin-member": "{{GENDER:$1|system administrator}}",
-	"grouppage-sysadmin": "m:System administrators",
+	"grouppage-sysadmin": "m:Tech:SRE Volunteers",
 	"group-interwiki-admin": "Interwiki Administrator",
 	"group-interwiki-admin-member": "interwiki administrator",
 	"grouppage-interwiki-admin": "m:Interwiki administrators",
 	"group-oversight": "Oversighters",
 	"group-oversight-member": "oversight",
 	"grouppage-oversight": "m:Oversight",
-	"group-checkuser": "Checkusers",
-	"group-checkuser-member": "checkuser",
-	"grouppage-checkuser": "m:CheckUser",
 	"group-wikicreator": "Wiki Creators",
 	"group-wikicreator-member": "{{GENDER:$1|wiki creator}}",
-	"grouppage-wikicreator": "m:Wiki creators",
+	"grouppage-wikicreator": "m:Meta:Wiki creators",
 	"group-wikimanager": "Wiki Managers",
 	"group-wikimanager-member": "{{GENDER:$1|wiki manager}}",
 	"grouppage-wikimanager": "m:Wiki managers",
 	"group-rollbacker": "Rollbackers",
-        "grouppage-rollbacker": "Project:Rollbackers",
+	"grouppage-rollbacker": "m:Special:MyLanguage/User groups#{{int:Group-rollbacker}}",
 	"group-rollbacker-member": "{{GENDER:$1|rollbacker}}",
 	"group-autopatrolled": "Autopatrolled users",
-        "grouppage-autopatrolled": "Project:Autopatrolled users",
+	"grouppage-autopatrolled": "m:Special:MyLanguage/User groups#{{int:Group-autopatrolled}}",
 	"group-autopatrolled-member": "{{GENDER:$1|autopatrolled}}",
 	"group-confirmed": "Confirmed users",
 	"group-confirmed-member": "{{GENDER:$1|confirmed user}}",
+	"grouppage-confirmed": "m:Special:MyLanguage/User groups#{{int:Group-autoconfirmed}}",
 	"group-globalsysop": "Global Sysops",
 	"group-globalsysop-member": "{{GENDER:$1|Global Sysop}}",
 	"grouppage-globalsysop": "m:Global Sysops",
```

Comment:
Special:MyLanguage?

Reply:
Not sure why line 119 is highlighted, but if you mean line 122, then yeah

Reply:
because line 119 is what you changed, the review is for line 122